.github/DISCUSSION_TEMPLATE/gateway.yml

```diff
@@ -1,10 +1,6 @@
 title: "[Cosmos chain onboarding proposal] "
 labels: ["gateway"]
 body:
-  - type: markdown
-    attributes:
-      value: |
-       "# Overview"
```

Comment:
Removing these also means the sections wont be broken up with headers anymore, intentional?

Reply:
The headers already don't show up properly, so my opinion is they aren't of much use anyways.